CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,37 @@
+# Important
+
+The Paradise GitHub is not exempt from community and server rules, especially rules 0, 1, and 4. An inability to abide by the rules on the GitHub will result in disciplinary action up to, or including, a repository ban.
```

Comment:
```suggestion
The Paradise GitHub is not exempt from Paradise Station community and server rules, especially rules 0, 1, and 4. An inability to abide by the rules on GitHub will result in disciplinary action up to, or including, a repository ban.
```

Reply:
Ill do the other change, but `the GitHub` makes more sense here, since it concerns this repo, not the entire platform GitHub